interface.html

```diff
@@ -21,6 +21,20 @@
         <a class="btn btn-primary show-email-provider">Configure Email Template</a>
       </div>
     </div>
+
+    <div class="form-group clearfix">
+      <div class="col-sm-12">
+        <div class="checkbox checkbox-icon">
+          <input type="checkbox" id="email-domain">
+          <label for="email-domain">
+            <span class="check"><i class="fa fa-check"></i></span> Enable domain white listing
+          </label>
+        </div>
+      </div>
+      <div class="col-sm-12 email-domains-input hidden">
```

Comment:
This should be wrapped around `<div class="form-group"></div>` the you won't need `.email-domains-input { margin-top: 10px; }` in `interface.css`.